Rover/defines.h

```diff
@@ -88,11 +88,11 @@ enum fs_ekf_action {
 
 #define DISTANCE_HOME_MINCHANGE 0.5f  // minimum distance to adjust home location
 
-enum pilot_steer_type_t {
-    PILOT_STEER_TYPE_DEFAULT = 0,
-    PILOT_STEER_TYPE_TWO_PADDLES = 1,
-    PILOT_STEER_TYPE_DIR_REVERSED_WHEN_REVERSING = 2,
-    PILOT_STEER_TYPE_DIR_UNCHANGED_WHEN_REVERSING = 3,
+enum class pilot_steer_type : int8_t {
```

Comment:
```suggestion
enum class PilotSteerType : int8_t {
```

Reply:
@peterbarker san.
I agree. I will change it to PilotSteerType.